staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -1670,31 +1671,6 @@ func (c *Cloud) InstanceShutdownByProviderID(ctx context.Context, providerID str
 	return false, nil
 }
 
-// InstanceMetadataByProviderID returns metadata of the specified instance.
-func (c *Cloud) InstanceMetadataByProviderID(ctx context.Context, providerID string) (*cloudprovider.InstanceMetadata, error) {
```

Comment:
Safe to remove since InstanceV2 is disabled https://github.com/kubernetes/kubernetes/blob/d3c1e81d5e6796db9389cd73073f48d6a37c4621/staging/src/k8s.io/legacy-cloud-providers/aws/aws.go#L1397-L1400